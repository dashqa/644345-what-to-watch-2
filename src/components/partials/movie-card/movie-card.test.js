import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from "./movie-card";

jest.mock(`../../partials/movie-card-top/movie-card-top.jsx`, () => `MovieCardTop`);
jest.mock(`../../partials/movie-card-bottom/movie-card-bottom.jsx`, () => `MovieCardBottom`);

const props = {
  movie: {
    backgroundColor: ``
  },
};

it(`Movie card correctly renders after relaunch`, () => {
  const tree = renderer.create(<MovieCard {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
