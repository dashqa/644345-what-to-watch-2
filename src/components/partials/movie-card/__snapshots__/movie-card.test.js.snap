// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie card correctly renders after relaunch 1`] = `
<section
  className="movie-card movie-card--full"
>
  <div
    className="movie-card__hero"
  >
    <MovieCardTop
      isMainPage={false}
      movie={Object {}}
    />
  </div>
  <MovieCardBottom
    movie={Object {}}
  />
</section>
`;
